aspnetcore/client-side/import-export-interop.md

```diff
@@ -0,0 +1,394 @@
+| `JSObject`                  | `Object`     |            | ✅                | ✅                    | ✅ |
+| `String`                    | `String`     |            | ✅                | ✅                    | ✅ |
+| `Object`                    | `Any`        |            | ✅                |                       | ✅ |
+| `Span<Byte>`                | `MemoryView` |            |                   |                        | |
+| `Span<Int32>`               | `MemoryView` |            |                   |                        | |
+| `Span<Double>`              | `MemoryView` |            |                   |                        | |
+| `ArraySegment<Byte>`        | `MemoryView` |            |                   |                        | |
+| `ArraySegment<Int32>`       | `MemoryView` |            |                   |                        | |
+| `ArraySegment<Double>`      | `MemoryView` |            |                   |                        | |
+| `Task`                      | `Promise`    |            |                   | ✅                    | |
+| `Action`                    | `Function`   |            |                   |                        | |
+| `Action<T1>`                | `Function`   |            |                   |                        | |
+| `Action<T1, T2>`            | `Function`   |            |                   |                        | |
+| `Action<T1, T2, T3>`        | `Function`   |            |                   |                        | |
+| `Func<TResult>`             | `Function`   |            |                   |                        | |
+| `Func<T1, TResult>`         | `Function`   |            |                   |                        | |
+| `Func<T1, T2, TResult>`     | `Function`   |            |                   |                        | |
+| `Func<T1, T2, T3, TResult>` | `Function`   |            |                   |                        | |
+
+The following conditions apply to type mapping and marshalled values:
+
+* The :::no-loc text="Array of"::: column indicates if the .NET type can be marshalled as a JS [`Array`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array). Example: C# `int[]` (`Int32`) mapped to JS `Array` of `Number`s.
+* When passing a JS value to C# with a value of the wrong type, the framework throws an exception in most cases. The framework doesn't perform compile-time type checking in JS.
+* `JSObject`, `Exception`, `Task` and `ArraySegment` create `GCHandle` and a proxy. You can trigger disposal in developer code or allow [.NET garbage collection (GC)](/dotnet/standard/garbage-collection/) to dispose of the objects later. These types carry significant performance overhead.
+* `MemoryView`
+  * `MemoryView` is a .NET type, not a native JS type. It's API is described in the [`dotnet.d.ts` file (`dotnet/runtime` GitHub repository)&dagger;](https://github.com/dotnet/runtime/blob/main/src/mono/wasm/runtime/dotnet.d.ts).
```

Comment:
I don't see any mention of MemoryView in dotnet.d.ts. I do see this: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSType.cs#L90

Reply:
I think it's https://github.com/dotnet/runtime/blob/release/7.0/src/mono/wasm/runtime/marshal.ts#L379

Reply:
Perhaps we should speak about `IMemoryView `
https://github.com/dotnet/runtime/blob/main/src/mono/wasm/runtime/marshal.ts#L436

@maraf do you remember if we had it in the `dotnet.d.ts` ? Should we add it ?

Reply:
I'm going to cross-link to `IMemoryView` (without the line number tho ... that's too fragile for us). I'm refactoring the prior remark about selecting the ref source branch for a given release. To harden the doc, it's best for us to link to `main` and let the dev find their version.